src/Catty/AskBrick+Instruction.swift

```diff
@@ -0,0 +1,65 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+extension AskBrick: CBInstructionProtocol {
+
+    @nonobjc func instruction() -> CBInstruction {
+        guard let question = self.question else { fatalError("Unexpected found nil.") }
+        guard let userVariable = self.userVariable else { fatalError("Unexpected found nil.") }
+
+        return CBInstruction.highPriorityExecClosure { context, _, _ in
+            Util.askUser(forTextAndPerformAction: #selector(self.callbackSubmit),
+                         target: self,
+                         cancelAction: #selector(self.callbackCancel),
+                         with: userVariable,
+                         promptTitle: nil,
+                         promptMessage: question.getDisplayString(),
```

Comment:
Please remove the quotes from the alert message:

![Screenshot 2020-09-29 at 07 10 13](https://user-images.githubusercontent.com/6848980/94514976-d45de900-0222-11eb-95ba-3df715eb636a.png)